sebgro98/Visitation-logger
# 12 frontend   create UI component log page

To test the implementation:
1. Navigate to http://localhost:5173/logs
2. Pressing "exportera" should download a logs.csv file which contains the logs in csv format, the filename should be concatenated with microseconds of the clock at that time
3. Pressing any of the buttons "Filtrera", "Föregående" and "Nästa" will print to the console to verify onclick functionality
